src/main/resources/static/templates/CSRFVulnerability/LEVEL_1/CSRFVulnerability.html

```diff
@@ -0,0 +1,30 @@
+<div id="CSRFVulnerability">
```

Comment:
you can reuse the HTML templates across levels so in case you are using the same js/CSS/HTML pages then I would suggest using the same templates. it will help reduce the duplicacy.